x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -213,7 +218,7 @@ export class ExecutionHandler<
         ruleRunMetricsStore.incrementNumberOfTriggeredActions();
         ruleRunMetricsStore.incrementNumberOfTriggeredActionsByConnectorType(actionTypeId);
 
-        if (isSummaryAction(action) && summarizedAlerts) {
+        if (summarizedAlerts) {
```

Comment:
just wondering why we no longer need this check for `isSummaryAction(action)`?

Reply:
I changed the Executable interface, from optional alert and summarizedAlerts to have at least one of them.
So now, having `summarizedAlerts` already means that the action is a summary action.